commit 4987a6b
Author: Robert Rouhani <robert.rouhani@gmail.com>
Date:   Fri Mar 20 23:51:24 2015 -0400

    Updated SharpFont to 3.0.0, adjusted font rendering code for new fixed-point types.
---
 OpenRA.Game/Graphics/SpriteFont.cs          |  4 ++--
 thirdparty/fetch-thirdparty-deps-windows.sh | 14 +++-----------
 thirdparty/fetch-thirdparty-deps.ps1        | 20 ++++++--------------
 thirdparty/fetch-thirdparty-deps.sh         |  5 +++--
 4 files changed, 14 insertions(+), 29 deletions(-)

diff --git OpenRA.Game/Graphics/SpriteFont.cs OpenRA.Game/Graphics/SpriteFont.cs
index 65a885e..5f79683 100644
--- OpenRA.Game/Graphics/SpriteFont.cs
+++ OpenRA.Game/Graphics/SpriteFont.cs
@@ -102,13 +102,13 @@ GlyphInfo CreateGlyph(Pair<char, Color> c)
 			face.LoadChar(c.First, LoadFlags.Default, LoadTarget.Normal);
 			face.Glyph.RenderGlyph(RenderMode.Normal);
 
-			var size = new Size((int)face.Glyph.Metrics.Width >> 6, (int)face.Glyph.Metrics.Height >> 6);
+			var size = new Size((int)face.Glyph.Metrics.Width, (int)face.Glyph.Metrics.Height);
 			var s = builder.Allocate(size);
 
 			var g = new GlyphInfo
 			{
 				Sprite = s,
-				Advance = (int)face.Glyph.Metrics.HorizontalAdvance / 64f,
+				Advance = (float)face.Glyph.Metrics.HorizontalAdvance,
 				Offset = new int2(face.Glyph.BitmapLeft, -face.Glyph.BitmapTop)
 			};
 
diff --git thirdparty/fetch-thirdparty-deps.sh thirdparty/fetch-thirdparty-deps.sh
index dcd4b9b..8452657 100755
--- thirdparty/fetch-thirdparty-deps.sh
+++ thirdparty/fetch-thirdparty-deps.sh
@@ -39,10 +39,11 @@ fi
 
 if [ ! -f SharpFont.dll ]; then
 	echo "Fetching SharpFont from nuget"
-	nuget install SharpFont -Version 2.5.0.1
-	cp ./SharpFont.2.5.0.1/lib/net20/SharpFont* .
-	sed '/osx/s@\(dll="\)[^"]*\(" />\)@\1/Library/Frameworks/Mono.framework/Libraries/libfreetype.6.dylib\2@' ./SharpFont.2.5.0.1/Content/SharpFont.dll.config > SharpFont.dll.config
-	rm -rf SharpFont.2.5.0.1
+	nuget install SharpFont -Version 3.0.1
+	cp ./SharpFont.3.0.1/lib/net20/SharpFont* .
+	cp ./SharpFont.3.0.1/config/SharpFont.dll.config .
+	rm -rf SharpFont.3.0.1
+	rm -rf SharpFont.Dependencies.2.5.5.1
 fi
 
 if [ ! -f nunit.framework.dll ]; then
